Cargo.toml

```diff
@@ -27,8 +27,8 @@ fixed_window_roller = []
 size_trigger = []
 time_trigger = ["rand"]
 onstartup_trigger = []
-json_encoder = ["serde", "serde_json", "chrono", "log-mdc", "log/serde", "thread-id"]
-pattern_encoder = ["chrono", "log-mdc", "thread-id"]
+json_encoder = ["serde", "serde_json", "chrono", "log-mdc", "log/serde"]
+pattern_encoder = ["chrono", "log-mdc"]
```

Comment:
i think actually to make this a no-breaking change, you will need to move these feature under a `[target.'cfg(not(target_family = "wasm"))'` and leave them as is, while also moving these variants that have thread-id removed under a `[target.'cfg((target_family = "wasm"))'`